File: src/main/resources/heartbeat_monitor/TitlePlugin.js
export function apply(title) {
    document.title = title;
    document.getElementById("topbar-right-container").prepend(
        also(document.createElement("div"), titleDiv => {
            titleDiv.className = "topbar-title";
            titleDiv.textContent = document.title;
        }),
    );
}

File: heartbeat_monitor/src/jsMain/kotlin/TitlePlugin.kt
package hello

import kotlinx.browser.document

object TitlePlugin : AbstractPlugin() {
    override suspend fun applyImpl() {
        document.title = "Kanban Bro Heartbeat Monitor"
        UiContainers.topbarRightContainer.prepend(
            document.createElement("div").also { titleDiv ->
                titleDiv.className = "topbar-title"
                titleDiv.textContent = document.title
            },
        )
    }
}
